print/components/PicassoChunk.vue

```diff
@@ -1,25 +1,25 @@
 <template>
   <div class="tw-break-after-page">
-    <div :class="landscape ? 'landscape' : ''">
+    <div :class="landscape ? 'landscape' : 'portrait'">
       <div class="tw-flex tw-flex-row tw-items-baseline fullwidth">
         <h1
           :id="`content_${index}_period_${period.id}`"
-          class="tw-text-2xl tw-font-bold tw-mb-6 tw-flex-grow tw-d-inline"
+          class="text-2xl-relative tw-font-bold tw-mb-1 tw-flex-grow tw-d-inline"
         >
           {{ $tc('print.picasso.title') }}
           {{ period.description }}
         </h1>
         <span>{{ camp.organizer }}</span>
         <img
           v-if="camp.printYSLogoOnPicasso"
-          height="35"
-          width="35"
+          :height="landscape ? 24 : 35"
+          :width="landscape ? 24 : 35"
           :src="ysLogoUrl"
           class="tw-self-start tw-ml-2"
         />
       </div>
 
-      <v-sheet :width="landscape ? 960 : 680">
+      <v-sheet width="680">
```

Comment:
is this on purpose the same value as the width for .landscape in the css?

why do we define the width here inline, and the other in the css?

Reply:
This width is not a CSS property but a property of the v-sheet component. It helps the v-calendar to render in the proper size.

Reply:
Oh, ok. thank you

Reply:
I tried setting directly from CSS and it still worked (https://github.com/ecamp/ecamp3/pull/3495/commits/ddc130919ee629bd9f33f54d5ff5ff9bb1296ae4)